src/config/constants.ts

```diff
@@ -0,0 +1,62 @@
+enum PLACEHOLDER_TEXT {
```

Comment:
use the following way of exporting all over the app:
`export enum PLACEHOLDER_TEXT {...`
also enums should be in types.ts file, constants.ts should contain only constants